restapi.go

```diff
@@ -472,6 +472,29 @@ func (s *Session) Guild(guildID string) (st *Guild, err error) {
 	return
 }
 
+// GuildWithCounts returns a Guild structure of a specific Guild with approximate member and presence counts.
+// guildID    : The ID of a Guild
+// withCounts : When true, will return approximate member and presence counts for the guild
+//              uses variadic for backwards compatibility
+func (s *Session) GuildWithCounts(guildID string, withCounts bool) (st *Guild, err error) {
```

Comment:
Since this function already retrieves guild with counts, I think it would be fair to drop `withCounts` parameter.